packages/ia-topnav/src/ia-topnav.js

```diff
@@ -172,6 +172,7 @@ export default class IATopNav extends LitElement {
         .baseHost=${this.baseHost}
         .config=${this.config}
         .open=${this.signedOutOpened}
+        ?hideSearch=${this.hideSearch}
```

Comment:
Q: Is this part of the fix or is this something else?

Reply:
Yes